realtime_ai_character/static/script.js

```diff
@@ -359,7 +359,6 @@ talkButton.addEventListener("click", function() {
     messageButton.style.display = "flex";
     stopCallButton.style.display = "flex";
     soundWave.style.display = "flex";
-    textContainer.textContent = "Hi, my friend, what brings you here today?";
```

Comment:
I think this can still be kept for visual prompt

Reply:
Done